src/user/bans.js

```diff
@@ -45,6 +45,9 @@ module.exports = function (User) {
 		if (until > now) {
 			tasks.push(async.apply(db.sortedSetAdd, 'users:banned:expire', until, uid));
 			tasks.push(async.apply(User.setUserField, uid, 'banned:expire', until));
+		} else {
+			tasks.push(async.apply(db.sortedSetRemove, 'users:banned:expire', uid));
+			tasks.push(async.apply(User.setUserField, uid, 'banned:expire', 0));
```

Comment:
heh, so we don't have a `removeUserField`, eh. Just want to make sure `banned:expire` handles `0`, because I also think `0` can be misinterpreted to mean permaban.

Reply:
I did think of that, and I initially used `db.deleteObjectField()`, but then I noticed that the [`unban`](https://github.com/NodeBB/NodeBB/blob/67a6fb70cd116156eff9f190093f975699a7773e/src/user/bans.js#L58) method does in fact set the `banned:expire` to `0` so.. I followed that logic.

It won't be interpreted as permaban unless `user.banned` is truthy